### DISCUSSION TITLE
Codepen to Github CSS not applying

### DISCUSSION THREAD

**Discussion by nac0512** [community/community#23734] [Other Feature Feedback, Questions, & Ideas]
<p>I’m having trouble getting my CSS from Codepen to apply to my Github Pages. I’m fairly new, but have looked at a TON of google searches and nothing seems to be helping.</p>
<p>This is what I have in my style.css</p>
<pre><code class="lang-css">html, 
body {
  font-family: system-ui;
  font-size: 20px;
  background: #000000;
  color: red;
  text-align: center;
}
h1 {
  color: blue;
}
</code></pre>
<p>This is what is returned. <a href="https://nac0512.github.io/test2/" rel="nofollow noopener">https://nac0512.github.io/test2/</a><br>
This is what it is supposed to look like on Codepen. <a href="https://codepen.io/nac0512/pen/MWKpJvb" rel="nofollow noopener">https://codepen.io/nac0512/pen/MWKpJvb</a></p>
<p>I’ve tried it both with and without the html tag.<br>
I have the &lt; !DOCTYPE html &gt; at the beginning of my index.html file.<br>
Is there something I’m missing? Any help is appreciated. I know it works because when I right click on the link from my exported zip file from Codepen, the site has all the formatting it’s supposed to. I’m literally dragging and dropping the files directly from desktop to my repo.</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Hey, welcome to the <strong>GitHub Support Community</strong>! The reason your CSS is not displaying properly is because your CSS stylesheet is not specified in the <code>head</code> element of your HTML file. This works on CodePen because CodePen automatically links your CSS and JavaScript to the HTML code on their site.</p>
<p>To fix this issue and make your CSS stylesheet work on your <strong>GitHub Pages</strong> site, place the following block of code after the <code>&lt;!DOCTYPE html&gt;</code> line in your HTML file:</p>
<pre><code class="lang-auto">&lt;head&gt;
    &lt;link rel="stylesheet" href="style.css"&gt;
&lt;/head&gt;
</code></pre>
<p>Hope this fixes your issue! If it doesn’t work please let me know, if it does then mark this reply as a solution!</p>

**Comment by nac0512** [comment]
<p>It worked, perfect! Thank you. For future reference, do I just have to put that at the beginning of every project I import?</p>

**Comment by Gisgar3** [comment]
<p>For every project you import from CodePen, you’ll need to add that entire <code>head</code> element block into your HTML file (just make sure to change the name of the CSS file in that block if it is different). Glad it worked out!</p>

**Comment by nac0512** [comment]
<p>Got it. Thanks so much.</p>
